qgis_deployment_toolbelt/profiles/profiles_handler_base.py

```diff
@@ -179,7 +179,7 @@ def _is_local_path_git_repository(
             local_path: Path = self.SOURCE_REPOSITORY_PATH_OR_URL
 
         try:
-            Repo(root=f"{local_path.resolve()}")
+            porcelain.open_repo_closing(f"{local_path.resolve()}")
```

Comment:
I can't remember when or why but I'm pretty sure we used porcelain until a certain issue that make us use the Repo class instead. If you can have a look to the git blame/history :pray:

Reply:
Nothing in blame. Created like this in local_git_handler then moved to base class.